src/main/java/com/scottescue/dropwizard/entitymanager/TemporaryClassLoader.java

```diff
@@ -1,156 +1,99 @@
 package com.scottescue.dropwizard.entitymanager;
 
 import com.google.common.annotations.VisibleForTesting;
+import javassist.ByteArrayClassPath;
+import javassist.ClassPool;
+import javassist.CtClass;
+import javassist.NotFoundException;
 import org.hibernate.engine.jdbc.StreamUtils;
 
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
-import java.io.OutputStream;
 
-class TemporaryClassLoader extends ClassLoader {
+/**
+ * ClassLoader implementation that allows classes to be temporarily loaded and then thrown away.
+ */
+public class TemporaryClassLoader extends ClassLoader {
```

Comment:
The class should have default access to prevent it form polluting the public API.